build/preprocessor.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+/* jshint node: true */
+/* global exports,
+          require */
+var utils = require('utils');
+
+const jsSuffix = /\.js$/;
+const htmlSuffix = /\.(html|htm)$/;
+
+function removeFiles(stagePath, list) {
+  list.forEach((fileName) => {
+    fileName.unshift(stagePath);
+    var file = utils.getFile.apply(this, fileName);
+    utils.log('PREPROCESSOR', 'remove file:', file.path);
+    if (utils.fileExists(file.path)) {
+      file.remove();
+    }
+  });
```

Comment:
```file.remove();``` is kind of a xpcshell only APIs,
try to use ```utils.deleteFile(path)``` with a given path.

start with line 14:
```js
var filePath = utils.joinPath.apply(this, fileName);
utils.log('PREPROCESSOR', 'remove file:', filePath);
utils.deleteFile(filePath);
```